tests/test_qt_js_integration.py

```diff
@@ -10,6 +10,9 @@
 import qmxgraph.mime
 
 
+m = pytest.mark.xfail(run=False, reason="Failing on GitHub actions, needs investigation")
```

Comment:
No idea why all tests in this module are crashing `pytest`... I only left `test_blank` running, and it still crashes.

Any ideas @prusse-martin ?

Reply:
The traceback point to `_qapp_instance = qt_api.QApplication(qapp_args)` in [pytestqt/plugin.py:57](https://github.com/pytest-dev/pytest-qt/blob/3.3.0/pytestqt/plugin.py#L57).
It looks like [github actions do not handle QT](https://github.community/t/feature-request-pre-installed-qt-on-windows/16207) out of the box.
Some people already created actions for that:
- https://github.com/jurplel/install-qt-action
- https://github.com/Skycoder42/action-setup-qt

Reply:
Interesting, but I don't think that's the issue, PyQt installs everything needed and https://github.com/pytest-dev/pytest-qt also uses GitHub actions and works just fine... definitely is something specific to our code, I think. 🤔

Reply:
Using that install Qt action just to make sure, let's see. 👍

Reply:
OH MY was I wrong. 😱 

It did work... now that it works, I'm guessing it is something that's installed on the system when Qt is installed, and is needed by the JS tests? Weird. Oh well. Thanks a lot @prusse-martin by the nudge!

Reply:
Contributors shall contribute.
😃